ctapipe/image/tests/test_hillas.py

```diff
@@ -3,26 +3,23 @@
 import numpy as np
 import pytest
 from astropy import units as u
-from astropy.coordinates import Angle, SkyCoord
+from astropy.coordinates import Angle
 from numpy import isclose
 from pytest import approx
 
-from ctapipe.containers import (
-    CameraHillasParametersContainer,
-    HillasParametersContainer,
-)
-from ctapipe.coordinates import TelescopeFrame
+from ctapipe.containers import HillasParametersContainer
+from ctapipe.coordinates import CameraFrame, TelescopeFrame
 from ctapipe.image import tailcuts_clean, toymodel
 from ctapipe.image.hillas import HillasParameterizationError, hillas_parameters
 from ctapipe.instrument import CameraGeometry, SubarrayDescription
 
 
 def create_sample_image(
     psi="-30d",
-    x=0.2 * u.m,
-    y=0.3 * u.m,
-    width=0.05 * u.m,
-    length=0.15 * u.m,
+    x=0.2 * u.deg,
```

Comment:
Did you check that these images make sense when just changing meters to degrees?

Reply:
I checked for the notebooks, I can recheck for all other files